jnigen/test/kotlin_test/kotlin/src/main/kotlin/com/github/dart_lang/jnigen/SuspendFun.kt

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
```

Comment:
2023 (also other files probably)

Reply:
What's the rule here? I probably added the files in 2022, it will be merged in 2023. Is it based on the time it will be merged to main or...?

Reply:
Squash on merge, so the commit will be 2023.